src/frontend/screens/Library/components/InstallModal/WineSelector/index.tsx

```diff
@@ -90,22 +88,14 @@ export default function WineSelector({
       {!useDefaultSettings && (
         <>
           {showPrefix && (
-            <TextInputWithIconField
+            <PathSelectionBox
+              type="directory"
+              onPathChange={setWinePrefix}
+              path={winePrefix}
+              pathDialogTitle={t('box.wineprefix', 'Select WinePrefix Folder')}
               label={t('install.wineprefix', 'WinePrefix')}
               htmlId="setinstallpath"
-              placeholder={winePrefix}
-              value={winePrefix.replaceAll("'", '')}
```

Comment:
is this call to `replaceAll` important? it's lost in the refactor

Reply:
I don't see why we would have this, as it doesn't actually change the underlying value (that is already stripped of some problematic characters when it's read from the settings) and besides, anything using it should be able to handle special characters anyways (at least everything that uses `spawn` instead of `exec`)

Reply:
yeah, I don't really know why this was added in the first place (maybe somebody knows of a specific game that needs it but I don't)